src/main/java/com/google/codeu/data/Location.java

```diff
@@ -29,9 +42,30 @@ public String getDescription() {
         return description;
     }
 
-    public UUID getID() {
+    public String getID() {
         return id;
     }
 
+    public String getImageURL() {
```

Comment:
Leave this to the person implementing the images part - I don't think you need it now?